File: src/main/java/ifexpression/java/IfExpression.java
package ifexpression.java;

import java.util.Random;

@SuppressWarnings("ALL")
public final class IfExpression {
    public static final void main() {
        int number = new Random().nextInt(10);
        String log = number < 3 && number > 1 ? "A" : (number > 8 ? "B" : (number == 1 ? "C" : "D"));
        System.out.println(log);
    }

    public static /* synthetic */ void main(String[] args) {
        main();
    }
}

File: src/main/kotlin/ifexpression/kotlin/IfExpression.kt
package ifexpression.kotlin

import java.util.*

@Suppress("ConvertTwoComparisonsToRangeCheck")
fun main() {
    val number: Int = Random().nextInt(10)
    val log: String = if (number < 3 && number > 1) {
        "A"
    } else if (number > 8) {
        "B"
    } else if (number == 1) {
        "C"
    } else {
        "D"
    }
    println(log)
}
